src/frontend/apps/desk/src/features/mail-domains/access-management/components/ChooseRole.tsx

```diff
@@ -1,65 +1,60 @@
 import { Radio, RadioGroup } from '@openfun/cunningham-react';
 import { useTranslation } from 'react-i18next';
 
-import { Role } from '../../domains';
+import { ORDERED_ROLES, Role } from '../../domains';
 
 interface ChooseRoleProps {
   availableRoles: Role[];
+  roleForSelectedUser: Role;
   currentRole: Role;
   disabled: boolean;
```

Comment:
Is `disabled` prop still necessary ?

Reply:
I asked myself the same question, I think it's cool to leave it, if one day we want to use this part in another form. Could be more complex. The logic is already there. I don't have a strong opinion on it so I'll let you decide.